actors/evm/tests/call.rs

```diff
@@ -1026,6 +1074,45 @@ impl From<CallActorReturn> for Vec<u8> {
     }
 }
 
+impl From<&[u8]> for CallActorReturn {
```

Comment:
added this back since it was useful a couple of times, It's not used in any tests ATM but its nice to leave here for now to see what the precompile is returning. It could also use a cleanup since we can just use `U256::from(x).as_u32()` to get values we need since we expect to panic otherwise. Later time though.